chartfx-math/src/main/codegen/de/gsi/math/MathGen.java

```diff
@@ -4890,7 +3580,7 @@ public static double VavilovI(double x, double kappa, double beta2) {
      * @param itype ???
      * @param npt ???
      */
-    public static void VavilovSet(double rkappa, double beta2, boolean mode, double[] WCM, double[] AC, double[] HC, int[] itype, int[] npt) {
+    public static void vavilovSet(double rkappa, double beta2, boolean mode, double[] WCM, double[] AC, double[] HC, int[] itype, int[] npt) {
```

Comment:
Codacy found an issue: [The method 'vavilovSet(double, double, boolean, double, double, double, int, int)' has an NPath complexity of 2880, current threshold is 200](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)